src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/eligibility/ObjectionEligibility.java

```diff
@@ -5,13 +5,13 @@
 public class ObjectionEligibility {
 
     @JsonProperty("is_eligible")
-    private boolean isEligible;
+    private boolean eligible;
 
-    public ObjectionEligibility(boolean isEligible) {
-        this.isEligible = isEligible;
+    public ObjectionEligibility(boolean eligible) {
+        this.eligible = eligible;
     }
 
     public boolean isEligible() {
```

Comment:
Jackson assumes this corresponds to a field called eligible, as there wasn't one it was adding an "eligible" field to the returned json.
Changing the field to eligible removes this issue

Reply:
Just interested, how come this worked before? Didn't it mean we'd send a field back to the client that wasn't recognised or is there a change to do in 'web' too?

Reply:
Jackson was sending two fields.
One it got from the JSON property annotation "is_eligible"
And another it got from inferring there should be a field called "eligible" from this getters name.

So the json response looked like
{
    is_eligible: true,
    eligible: true
}

I think the web was just ignoring the other field

Reply:
Ah right 👍 .